### DISCUSSION TITLE
Correction for words with very short or negative duration in the Highlight_Words option

### DISCUSSION THREAD

**Discussion by Joelson-Forte** [Purfview/whisper-standalone-win#274] [Q&A]
@Purfview 
I use Whisper Highlight_Words option a lot to generate subtitles with underlined words '--highlight_words true --max_line_width 43 --max_line_count 2', but Whisper transcription generates some words with very short duration (less than 23ms or with a negative value) . Words with 23ms or a negative value do not correspond to even 1 video frame and therefore these words are not imported into the Timeline.

![2024-06-27_19h33_11](https://github.com/Purfview/whisper-standalone-win/assets/131564527/ecab4ee4-473d-49fe-a77f-47f7a7112e45)

Is there a feature in Faster-Whisper that fixes this problem? I did some tests and realized that if very short or negative values ​​are somehow corrected to 50ms, the problem is solved. I remember encountering a similar problem with the "One_Word" option but you managed to solve it. https://github.com/Purfview/whisper-standalone-win/discussions/107

**Comment by Purfview** [comment]
I don't have much interest in highlight_words nor in deciphering that code.
It's written by @JonasCz https://github.com/Purfview/whisper-standalone-win/discussions/229 , maybe he can help you.

**Comment by Purfview** [comment]
@JeditPro 
> I use Whisper... ...or with a negative value...

Wait, do you use Whisper or Faster-Whisper? And what version? I don't know how negative values are possible.
Share `json` file with such values and full command, then maybe I'll look at it.
